pokemongo_bot/cell_workers/recycle_items.py

```diff
@@ -3,10 +3,13 @@
 from pokemongo_bot.base_task import BaseTask
 from pokemongo_bot.tree_config_builder import ConfigException
 
+
 class RecycleItems(BaseTask):
     SUPPORTED_TASK_API_VERSION = 1
 
     def initialize(self):
+        self.min_inventory_items_to_run = self.config.get('min_inventory_items_to_run', None)
+        self.run_when_storage_less_than = self.config.get('run_when_storage_less_than', None)
```

Comment:
I don't think you need both config options. I'd be fine with just `min_empty_space`. If you feel like you really want to support both, we should probably throw if both are set.